core/build.gradle.kts

```diff
@@ -1,13 +1,16 @@
 import com.codingfeline.buildkonfig.compiler.FieldSpec.Type.STRING
 
 plugins {
-    id("revenuecat-library")
+    id("revenuecat-public-library")
     alias(libs.plugins.kotlin.cocoapods)
     alias(libs.plugins.codingfeline.buildkonfig)
 }
 
 kotlin {
     sourceSets {
+        commonMain.dependencies {
```

Comment:
Hmm I'm just wondering, since we will be exposing mapping functions for the android/ios source sets, should we depend on the mappings module for the android/ios source sets only? That way, we keep the `common` source set totally independent of the PHC classes?

Reply:
Great idea! Done here: https://github.com/RevenueCat/purchases-kmp/pull/157/commits/fee60185d03dea5858ccc6efe9ac4c6c976ae3b0 
  
(☝️ also regenerated `revenuecatui.podspec`. Apparently I didn't sync before my last commit.)